build/Jenkinsfile

```diff
@@ -128,23 +160,25 @@ podTemplate(label: 'jenkins-slave',
     }
     stage('Style Checks') {
         container('golang') {
-            sh 'cd /go/src/github.com/mattermost/mattermost-server && make check-style BUILD_NUMBER=$BUILD_NUMBER'
+            sh "cd /go/src/github.com/mattermost/mattermost-server && make check-style BUILD_NUMBER=${env.BUILD_NUMBER}"
```

Comment:
It would be nice to remove all of the `BUILD_NUMBER` passing from commands that don't use it. `make check-style` for example, does not use the build number in any way.

Reply:
done